corehq/apps/sms/api.py

```diff
@@ -41,6 +41,10 @@ class BackendAuthorizationException(Exception):
     pass
 
 
+def get_utcnow():
```

Comment:
in this case, it's a bit more complicated than that example because the code also checks isinstance(xyz, datetime) which would fail under those circumstances.  i found some posts online describing a way to handle that scenario but this seems like the clearest, easiest way to do it